### PR TITLE
Update deps grunt zanata js

### PR DESCRIPTION
**Pull request by tagoh**
There was a bug in grunt-zanata-js and was caused a build issue. this PR is to update the deps of grunt-zanata-js which contains a fix for that.

### CODE REVIEW

**Comment by corinnekrych** [comment]
@tagoh that does the trick!
+1

**Comment by wtrocki** [comment]
Reviewed and verified locally. Thanks for resolving this!

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "grunt-open": "0.2.3",
     "grunt-plato": "1.0.0",
     "grunt-shell": "0.6.4",
-    "grunt-zanata-js": "1.0.1",
+    "grunt-zanata-js": "^1.1.0",
```
Comment (tagoh) [inline package.json]: Ah, this line shouldn't contain ^

**Comment by tagoh** [comment]
Thanks. should I merge this by myself? or wait for someone else who is supposed to do so?

**Comment by tagoh** [comment]
Thanks!

**Pull request merged**